templates/careers/company-culture/remote-work.html

```diff
@@ -18,100 +18,117 @@
-        </p>
-      </div>
-    </div>
-  </section>
-
-  <section class="p-section">
-    <div class="u-fixed-width">
-      <hr class="p-rule" />
-    </div>
-    <div class="row">
-      <div class="col-6 col-medium-3">
-        <h2>We build trust</h2>
-      </div>
-      <div class="col-6 col-medium-3">
-        <p>
-          Zero commute means more time for the things you enjoy. At the same time, we have very high expectations of commitment and independent execution.
-        </p>
-        <p>We invest in people who are self-driven, motivated, and highly organized.</p>
-        <p>
-          We take our responsibility for the most critical elements of open infrastructure very seriously, and we are committed to quality in every aspect of our product and services.
-        </p>
-        <p>
-          <a href="/careers/company-culture">Read more about our tech culture&nbsp;&rsaquo;</a>
-        </p>
-      </div>
-    </div>
-  </section>
-
-  <section class="p-section">
-    <div class="u-fixed-width">
-      <hr class="p-rule" />
-    </div>
-    <div class="row">
-      <div class="col-6 col-medium-3">
-        <h2>We engineer excellence</h2>
-      </div>
-      <div class="col-6 col-medium-3">
-        <p>
-          We aspire to lead on the global stage. There’s no other way to describe it than hard work, hard challenges and hard competition.
-        </p>
-        <p>
-          You will need to show that you can compete in deeply challenging intellectual fields. Your colleagues are aiming for the top, and they depend on you to set the same standard in your area of responsibility. That’s tough but incredibly satisfying. We value those who take ownership of their area and take opportunities to be exceptional.
-        </p>
-        <p>
-          <a href="/careers/career-explorer">What kind of excellent are you&nbsp;&rsaquo;</a>
-        </p>
+        <div class="p-strip is-shallow u-no-padding--top">
+          <h2>We are remote but connected</h2>
+        </div>
+        <div class="p-section--shallow">
+          <p>Canonical has been a fully distributed organization since its origin in 2004.</p>
+          <p>
+            Our remote-first policy enables us to collaborate with the best in the world in our domain. Though most team members work remotely, we meet each other at least twice per year during in-person sprints to nurture cross-team relationships. These sprints are funded by Canonical, and take place at locations around the world.
+          </p>
+        </div>
+        <hr class="p-rule--muted" />
+        <a class="p-button--positive u-no-margin--bottom" href="/careers">
+          Careers with Canonical
+        </a>
       </div>
     </div>
   </section>
 
-  <section class="p-section">
+  <section class="p-strip u-no-padding--top">
```

Comment:
```suggestion
  <section class="p-section">
```
We use p-strip class on sections only if they have a different background, as mentioned in the [docs](https://vanillaframework.io/docs/patterns/section#regular-sections).
Please use `p-section` here, which will also eliminate top padding.